src/aat/resources/features/F-105/F-105.feature

```diff
@@ -0,0 +1,201 @@
+    And the request [contains a proper Case Role CR-1 in one entry and no Case Role in the other]
+    And it is submitted to call the [Add Case-Assigned Users and Roles] operation of [CCD Data Store api],
+    Then a negative response is received,
+    And the response has all the details as expected.
+    And a call [to verify Olawale's reception of the role CR-1 over the case C1] will get the expected response as in [S-105.8_Get_Case_Roles_for_Case_C1].
+
+  # RDM-8606 no list
+  @S-105.9
+  Scenario: Must return an error response for missing case_users list
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Dil - who is to add some case role assignment for a case],
+    When a request is prepared with appropriate values,
+    And the request [is made from an authorised application by Dil, with no case_users supplied]
+    And it is submitted to call the [Add Case-Assigned Users and Roles] operation of [CCD Data Store api],
+    Then a negative response is received,
+    And the response has all the details as expected.
+
+  # RDM-8606 empty list
+  @S-105.10
+  Scenario: Must return an error response for empty case_users list
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Dil - who is to add some case role assignment for a case],
+    When a request is prepared with appropriate values,
+    And the request [is made from an authorised application by Dil, with an empty list of case_users supplied]
+    And it is submitted to call the [Add Case-Assigned Users and Roles] operation of [CCD Data Store api],
+    Then a negative response is received,
+    And the response has all the details as expected.
+
+  # RDM-8606 case not found
+  @S-105.11
+  Scenario: Must return an error response when the case does not exist
+    Given an appropriate test context as detailed in the test data source,
+    And a user [Richard - who can create a case],
+    And a user [Dil - who is to add some case role assignment for a case],
+    And a user [Olawale - with an active solicitor profile and valid User ID],
+    And a successful call [by Richard to create a case - C1] as in [F-105_Prerequisite_Case_Creation_Call_for_Case_Assignment],
+    When a request is prepared with appropriate values,
+    And the request [is made by Dil for 2 assignments each containing Olawale's User ID and a proper Case Role CR-1]
+    And the request [contains the Case ID of C1 in one entry and a well formed but non-existant case ID in the other]
+    And it is submitted to call the [Add Case-Assigned Users and Roles] operation of [CCD Data Store api],
+    Then a negative response is received,
+    And the response has all the details as expected.
+    And a call [to verify Olawale's reception of the role CR-1 over the case C1] will get the expected response as in [S-105.11_Get_Case_Roles_for_Case_C1].
```

Comment:
a call [to verify that Olawale hasn't received the role CR-1 over the case C1] ...

Reply:
I have updated along with corresponding verify JSON.